homework3/task3.js

```diff
@@ -0,0 +1,24 @@
+function splitDelimitedString(delimitedString) {
+  if (typeof delimitedString === "string") {
+    const newArray = delimitedString
+      .replace("“", "")
```

Comment:
don't need to replace it. Because ` “ ”` it quotes from google doc. Your string is `"Red, Green, Blue, 1, White, 3, 4, 5, 7"`

Reply:
Ohh.. thanks .. thinking it was part of the task…